Lib/test/test_capi/test_getargs.py

```diff
@@ -531,29 +531,31 @@ def __bool__(self):
         raise NotImplementedError
 
 class Boolean_TestCase(unittest.TestCase):
-    def test_p(self):
-        from _testcapi import getargs_p
-        self.assertEqual(0, getargs_p(False))
-        self.assertEqual(0, getargs_p(None))
-        self.assertEqual(0, getargs_p(0))
-        self.assertEqual(0, getargs_p(0.0))
-        self.assertEqual(0, getargs_p(0j))
-        self.assertEqual(0, getargs_p(''))
-        self.assertEqual(0, getargs_p(()))
-        self.assertEqual(0, getargs_p([]))
-        self.assertEqual(0, getargs_p({}))
-
-        self.assertEqual(1, getargs_p(True))
-        self.assertEqual(1, getargs_p(1))
-        self.assertEqual(1, getargs_p(1.0))
-        self.assertEqual(1, getargs_p(1j))
-        self.assertEqual(1, getargs_p('x'))
-        self.assertEqual(1, getargs_p((1,)))
-        self.assertEqual(1, getargs_p([1]))
-        self.assertEqual(1, getargs_p({1:2}))
-        self.assertEqual(1, getargs_p(unittest.TestCase))
-
-        self.assertRaises(NotImplementedError, getargs_p, Paradox())
+    def test_p_and_P(self):
+        from _testcapi import getargs_p as _getargs_p, getargs_P
+
+        for getargs_p in (_getargs_p, getargs_P):
```

Comment:
```suggestion
        import _testcapi

        for parsefn in (_testcapi.getargs_p, _testcapi.getargs_P):
```

+ `with self.subTest(parsefn=parsefn)` so that we know which function is the culprit if needed.

Reply:
Added it